Uchu.World/Handlers/GameMessages/GeneralHandler.cs

```diff
@@ -24,12 +24,18 @@ public async Task RequestUseHandler(RequestUseMessage message, Player player)
                 // Multi-interact is mission
                 //
 
-                if (message.MultiInteractType == default)
+                if (message.MultiInteractType == 0) // Mission Component
```

Comment:
This one is Mission component but I don't know if it could also be something else. TODO: Investigate